docs/en/sql_reference/statements/alter.md

```diff
@@ -500,4 +500,92 @@ A mutation query returns immediately after the mutation entry is added (in case
 
 Entries for finished mutations are not deleted right away (the number of preserved entries is determined by the `finished_mutations_to_keep` storage engine parameter). Older mutation entries are deleted.
 
+## ALTER USER {#alter-user-statement}
+
+Changes ClickHouse user accounts.
+
+### Syntax {#alter-user-syntax}
+
+``` sql
+ALTER USER [IF EXISTS] name [ON CLUSTER]
+    [RENAME TO new_name]
+    [IDENTIFIED [WITH {PLAINTEXT_PASSWORD|SHA256_PASSWORD|DOUBLE_SHA1_PASSWORD}] BY {'password'|'hash'}]
+    [[ADD|DROP] HOST {LOCAL | NAME 'name' | NAME REGEXP 'name_regexp' | IP 'address' | LIKE 'pattern'} [,...] | ANY | NONE]
+    [DEFAULT ROLE role [,...] | ALL | ALL EXCEPT role [,...] ]
+    [SETTINGS variable [= value] [MIN [=] min_value] [MAX [=] max_value] [READONLY|WRITABLE] | PROFILE 'profile_name'] [,...]
+```  
+
+### Description {#alter-user-dscr}
+
+To use `ALTER USER` you must have the [CREATE USER](grant.md#grant-create-user) privilege.
+
+### Examples {#alter-user-examples}
+
+Set assigned roles as default:
+
+``` sql
+ALTER USER user DEFAULT ROLE role1, role2
+```
+
+If roles aren't previously granted to a user, ClickHouse throws an exception.
```

Comment:
Add

``` sql
ALTER USER user DEFAULT ALL
```
sets that all granted roles to user are default. If a role will be granted to user in future it will become default automatically.